### PR TITLE
RFC: db:seed and db:unseed

### PR DESCRIPTION
**Pull request by Americas**
Based on Issue #109, I have done what you suggested and added commands for seeding and unseeding. It is not production ready, and there are no tests, but I wanted you to check it out and tell me your thoughts. I realize some parts are a little bit hammered on, especially the umzug-helper.js code.

### CODE REVIEW

**Comment by sdepold** [comment]
Could you please rebase?

**Comment by Americas** [comment]
rebased

File: lib/tasks/db.js [hunk 1]
```diff
@@ -37,13 +37,63 @@ module.exports = {
...
+    task: function () {
+      getMigrator('seeder', function (migrator) {
+        migrator.executed()
+        .then(function (seeders) {
+          if (seeders.length === 0) {
+            console.log('No executed seeders found.');
+            process.exit(0);
+          }
+        }).then(function () {
+          return migrator.down({to: 0});
```
Comment (sdepold) [inline lib/tasks/db.js]: Is there a specific reason for this to undo all the seeding migrations at once? shouldn't it behave as `db:migrate:undo` or should we add `db:seed:undo`?

**Comment by sdepold** [comment]
Could you add some tests for this?

**Comment by Americas** [comment]
This was an RFC. I will work on creating tests for this then.

As for the unseed command, good catch. I will change it to be as the migrate command, with an undo and undo:all options

**Comment by sdepold** [comment]
:+1:

**Comment by Americas** [comment]
Added seed commands:
* db:seed - seeds the database
* db:seed:undo - undo the last seeding
* db:seed:undo:all - undo all seeds
* seed:create - command to generate seed files

the test is failing in 2 places, but when I try to replicate in my machine, using the versions that the CI is complaining about, it runs all the test correctly..

**Comment by sdepold** [comment]
Can you please rebase the PR?

**Comment by ataube** [comment]
Hi - can I help you out to get this PR merged? What is missing?

**Comment by jonhkr** [comment]
ping

**Comment by sdepold** [comment]
> What is missing?

time. I will try to look into this PR tomorrow! promised!

**Comment by ataube** [comment]
I also started looking at it last night, I was able to merge with master but there are some failing seeding tests. I guess something went wrong while resolving conflicts etc...
I can continue fixing the tests or you continue. How do we proceed?

**Comment by ataube** [comment]
I made a new working PR for this feature, please have a look at #176 and close this one.

**Comment by sdepold** [comment]
Closing this in favor of #176. Thanks @ataube and @Americas

**Pull request closed without merging**